### PR TITLE
Fix for purging indexed tables

### PR DESCRIPTION
**Pull request by Methma**
## Purpose
Modifications for delete operations in indexing tables. Fix for <DAS_HOME>/repository/data directory growth.

### CODE REVIEW

File: components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java [hunk 1]
```diff
@@ -60,35 +52,35 @@
...
-        logger.info("Starting Test testMultipleRecordStores");
-        this.cleanupTable(1, "T1");
+        this.cleanupTable(1, "T1");        
         List<String> recordStoreNames = this.service.listRecordStoreNames();
         Assert.assertTrue(recordStoreNames.size() > 0);
         this.service.createTable(1, "T1");
         Assert.assertTrue(recordStoreNames.contains(this.service.getRecordStoreNameByTable(1, "T1")));
         if (recordStoreNames.size() > 1) {
-            logger.debug("** Multiple Record Stores Found **");
+            System.out.println("** Multiple Record Stores Found **");
```
Comment (tishan89) [inline components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java]: Please remove sys out

File: components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java [hunk 2]
```diff
@@ -688,48 +690,47 @@ public void testRecordAddRetrieveWithKeyValues() throws AnalyticsException {
...
-    @Test(dependsOnMethods = "testRecordAddRetrieveWithKeyValues")
+    
+    @Test (dependsOnMethods = "testRecordAddRetrieveWithKeyValues")
     public void testDataRecordAddReadPerformanceNonIndex() throws AnalyticsException {
-        logger.info("Starting Test testDataRecordAddReadPerformanceNonIndex");
         int tenantId = 51;
         String tableName = "TableX";
         this.cleanupTable(tenantId, tableName);
-        logger.info("\n************** START ANALYTICS DS (WITHOUT INDEXING) PERF TEST **************");
+        System.out.println("\n************** START ANALYTICS DS (WITHOUT INDEXING) PERF TEST **************");
```
Comment (tishan89) [inline components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java]: Please remove sout

Comment (tishan89) [reply components/analytics-core/org.wso2.carbon.analytics.datasource.core.test/src/main/java/org/wso2/carbon/analytics/datasource/core/AnalyticsDataServiceTest.java]: Applies to all below occurrences

**Comment by Methma** [comment]
Closing this PR since there are changes from the 1.2.7 branch.

**Pull request closed without merging**